pkg/jmxfetch/jmxfetch.go

```diff
@@ -165,6 +166,11 @@ func (j *JMXFetch) setDefaults() {
 	if j.Output == nil {
 		j.Output = log.JMXInfo
 	}
+	if j.JavaOptions == "" {
+		j.JavaOptions = jmxAllowAttachSelf
+	} else if !strings.Contains(j.JavaOptions, strings.TrimSpace(jmxAllowAttachSelf)) {
```

Comment:
I think this is good to check, but I'm curious if there is any negative impact to having this option specified twice. I'm going to guess no, but curious if there was a particular motivation for this.

Reply:
There is no negative impact which is why i was debating not having these checks